frontend/src/static/js/components/webstatus-line-chart-panel.ts

```diff
@@ -304,13 +304,23 @@ export abstract class WebstatusLineChartPanel extends LitElement {
     ];
   }
 
+  /**
+   * Update the formatted chart data.
+   * @param dataObj The new formatted chart data.
+   */
+  updateData(dataObj: WebStatusDataObj) {
+    this.data = dataObj;
+  }
+
```

Comment:
```suggestion
```

Reply:
I don't think this is used anymore.

Reply:
Yes, good catch. This was planned to be used in `_populateDataForChart`, but was discarded after another approach.